src/graph/FetchVerticesExecutor.cpp

```diff
@@ -92,6 +97,23 @@ Status FetchVerticesExecutor::prepareVids() {
     return Status::OK();
 }
 
+Status FetchVerticesExecutor::checkTagProps() {
+    auto aliasProps = expCtx_->aliasProps();
```

Comment:
We'd better add a validate function inside Expression to do that.

Reply:
The `AliasProps` have different meanings in different statements. We don't know if it is a tag or edge., so we can't validate it inside the expression.